lib/internal/bootstrap/node.js

```diff
@@ -213,6 +213,26 @@ const timers = require('timers');
 defineOperation(globalThis, 'clearImmediate', timers.clearImmediate);
 defineOperation(globalThis, 'setImmediate', timers.setImmediate);
 
+const { navigator, Navigator } = require('internal/navigator');
+
+// https://html.spec.whatwg.org/multipage/system-state.html#the-navigator-object
+ObjectDefineProperty(globalThis, 'navigator', {
+  __proto__: null,
+  enumerable: true,
+  configurable: true,
+  writable: false,
+  value: navigator,
+});
+
+// https://html.spec.whatwg.org/multipage/system-state.html#the-navigator-object
+ObjectDefineProperty(globalThis, 'Navigator', {
+  __proto__: null,
+  enumerable: true,
+  configurable: true,
+  writable: false,
+  value: Navigator,
+});
```

Comment:
Shouldn't this be moved to `lib/internal/bootstrap/web/exposed-window-or-worker.js`?

Reply:
I think you're right. I'm updating the pull request. Thank you for the review. Appreciate your time.